src/mongocxx/exception/error_code.hpp

```diff
@@ -33,6 +33,7 @@ enum class error_code : std::int32_t {
     k_ssl_not_supported,
     k_unknown_read_concern,
     k_unknown_write_concern,
+    k_server_response_malformed,
```

Comment:
Please update error_code.cpp with a string message as well.
(While you're at it, please sort the messages in the same order as the .hpp file)

I just added a minor comment on the 3.1 dev branch about this. Hopefully that won't cause a merge conflict.